### ISSUE TITLE
DDNS stopped working

### ISSUE THREAD

**Issue by cat-weazle** [dadatuputi/bitwarden_gcloud#72]
Since a while I am getting an insecure connection warning when opening my Vaultwarden vault using Chrome. In the logs of the DDNS container I see this message:
`ddns            | Invalid version format (non-numeric data) at /usr/bin/ddclient line 24.`
How do I fix this?

**Comment by cat-weazle** [comment]
/usr/bin/ddclient contains the following line:
`my $version = $VERSION->stringify();`

**Comment by asardaes** [comment]
That's a ddclient issue, they released a fix yesterday in v3.11.1 - https://github.com/ddclient/ddclient/issues/579

That version is just giving errors though - "could not determine an ip". I downgraded back to 3.10.0 for now.

**Comment by dadatuputi** [comment]
> Since a while I am getting an insecure connection warning when opening my Vaultwarden vault using Chrome. In the logs of the DDNS container I see this message: `ddns | Invalid version format (non-numeric data) at /usr/bin/ddclient line 24.` How do I fix this?

This sounds like two problems - a ddns error wouldn't have anything to do with getting cert warnings in Chrome. 

I don't have that error, I have manually pulled the latest images using `docker-compose pull` and then restart with `docker-compose down` and `docker-compose up -d`. The linuxserver/ddclient image was updated 2 days ago. Unfortunately I don't have old logs to see if I had that error previously. 

You shouldn't be getting cert errors, caddy should take care of keeping your certs up to date.

**Comment by dadatuputi** [comment]
> That's a ddclient issue, they released a fix yesterday in v3.11.1 - [ddclient/ddclient#579](https://github.com/ddclient/ddclient/issues/579)
> 
> That version is just giving errors though - "could not determine an ip". I downgraded back to 3.10.0 for now.
> 
> EDIT: 3.10.0 is also giving me errors due to connection refused, not sure what's going on.

How did you downgrade ddclient?

**Comment by dadatuputi** [comment]
> I modified the proxy's image in `docker-compose.yml` and hard-coded the tag to 3.10.0. Then I pulled and restarted with `docker-compose`.
> 
> BTW, I had been using `ssl=yes` in `ddclient.conf` and checking against `checkip.dyndns.org`. If I now change to `ssl=no`, it works fine, even with v3.11.1

Glad to hear it's working. Strange that it's related to that ssl config setting. I'm working fine on `latest` with `ssl=yes`. You also have the latest pull from this repo?

**Comment by asardaes** [comment]
Yes, I pulled it today when I noticed I had the same issue as cat-weazle. Even `curl -v https://checkip.dyndns.org:443` fails, so I guess it's on their side?

**Comment by dadatuputi** [comment]
Thanks for the follow-up @asardaes, that's a reasonable guess as to why ddclient was failing. Strange error message that doesn't seem to have anything to do with it, but glad it's working now! I'll close this as resolved.

**Comment by cat-weazle** [comment]
> > Since a while I am getting an insecure connection warning when opening my Vaultwarden vault using Chrome. In the logs of the DDNS container I see this message: `ddns | Invalid version format (non-numeric data) at /usr/bin/ddclient line 24.` How do I fix this?
> 
> This sounds like two problems - a ddns error wouldn't have anything to do with getting cert warnings in Chrome. Edit: unless it's pointing to an old address / not updating.
> 
> I don't have that error, I have manually pulled the latest images using `docker-compose pull` and then restart with `docker-compose down` and `docker-compose up -d`. The linuxserver/ddclient image was updated 2 days ago ([ddclient/ddclient#579 (comment)](https://github.com/ddclient/ddclient/issues/579#issuecomment-1779944236)). Unfortunately I don't have old logs to see if I had that error previously.
> 
> You shouldn't be getting cert errors, caddy should take care of keeping your certs up to date.

Thanks for your help @asardaes  and @dadatuputi. Pulling the latest images and restarting the containers fixed both issues for me.
